Steam Web Integration.user.js

```diff
@@ -317,13 +318,13 @@ function refreshBundles(callback) {
 function doApp(elem, wishlist, ownedApps, ignoredApps, decommissioned, limited, cards, bundles, dlc, lcs, dlcs, dlclcs, llcs, clcs, blcs) {
     $(elem).addClass(`swi`);
 
-    const attr = settings.attributes.find((a) => /apps?\/[0-9]+/g.test($(elem).attr(a)));
+    const attr = settings.attributes.find((a) => /a(pps?)?\/[0-9]+/g.test($(elem).attr(a)));

```

Comment:
did u test this on all link types?

Reply:
![image](https://user-images.githubusercontent.com/5698157/111019792-5bcb9d00-83fc-11eb-8ba9-f66fc233016c.png)

Reply:
![image](https://user-images.githubusercontent.com/5698157/111019808-76057b00-83fc-11eb-9aaa-3a1872834d35.png)